exporter/signalfxexporter/correlation/correlation.go

```diff
@@ -135,10 +129,13 @@ func (cor *Tracker) AddSpans(ctx context.Context, traces pdata.Traces) error {
 }
 
 // Start correlation tracking.
-func (cor *Tracker) start() {
-	if cor != nil && cor.correlation != nil {
-		cor.correlation.Start()
+func (cor *Tracker) Start(_ context.Context, host component.Host) (err error) {
```

Comment:
Why exposing the `Start` func?

Reply:
This is located in a separate package ('correlation') than the SignalFX Factory ('signalfxexporter').

Reply:
To see if I got this right: basically, the old start was called by the factory function in this source file, and it was OK to be private. Now, it's an appropriate component `Start` func, called by the component factory, located in another package.

Reply:
That's correct, @jpkrohling! The original code was interesting as the correlation tracker delayed being started until 'AddSpans' was called once.